test/integration/stats_integration_test.cc

```diff
@@ -214,8 +214,9 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithStats) {
   // 2019/04/27  6733     50213               Use SymbolTable API for HTTP codes
   // 2019/05/31  6866     50157               libstdc++ upgrade in CI
   // 2019/06/03  7199     49393               absl update
+  // 2019/06/06  7208     49419               make memory targets approximate
 
-  EXPECT_EQ(m_per_cluster, 49393);
+  EXPECT_LE(m_per_cluster, 49449);
```

Comment:
We will miss some regression on this since PRs saving memory likely won't update the value here. I don't really have idea to make this better while make it robust. Shall we do some exact match when it is running in our CI image only?

Reply:
Sure but I think that can be a follow-up. Maybe we could have a test-helper method that took the exact number we scrape from CI, but did an inexact comparison when not running in CI.

Reply:
SGTM, can you file an issue?